scss/components/colors.scss

```diff
@@ -0,0 +1,5 @@
+$sheetLighterBackground: #f6f5f7;
```

Comment:
This would be the 3rd colour file which seems a little excessive. Any thoughts on consolidation as I'm sure we'd like to use these colours elsewhere.

Reply:
I moved the colors into utils/_colors.scss and change their naming scheme to kebab-case.